config/config.php

```diff
@@ -12,24 +12,15 @@
  * Obviously, if you use closures in your config you can't cache it.
  */
 
-$cachedConfigFile = 'data/cache/app_config.php';
+$configManager = new ConfigManager([
+    // Load config from all modules first
+    new PhpFileProvider('src/*/Config/{{,*.}global,{,*.}local}.php'),
```

Comment:
IMO with natural structure for module is this:

```
MyModule/
    config/
    src/
    templates/
```

Every related file (classes, templates, configs) is grouped together under common directory. This is something people are already familiar with.